services/src/main/java/org/keycloak/protocol/oidc/OIDCClientSecretConfigWrapper.java

```diff
@@ -4,13 +4,15 @@
 import java.text.SimpleDateFormat;
 import java.util.HashMap;
 import java.util.Map;
+import java.util.Set;
+import java.util.stream.Stream;
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.keycloak.common.util.Time;
-import org.keycloak.models.ClientModel;
-import org.keycloak.models.ClientSecretConstants;
+import org.keycloak.models.*;
```

Comment:
Minor: Is it possible to remove star imports from the PR? And ideally set your IDE to avoid that? I am not 100% sure, but maybe it is included in the Intellij Idea style I shared some time ago?

I agree that avoiding "star" imports is probably not documented anywhere, but in general, we try to avoid that. I think it will be good to discuss a bit more about some more guidelines and making sure that these rules arr documented somewhere.

Reply:
Of course, I used the formatter but I don't know why it generated this import of "*".